test/functional/apps/context/index.ts

```diff
@@ -30,10 +30,12 @@ export default function ({ getService, getPageObjects, loadTestFile }: FtrProvid
     });
 
     loadTestFile(require.resolve('./_context_navigation'));
+    loadTestFile(require.resolve('./classic/_context_navigation'));
     loadTestFile(require.resolve('./_discover_navigation'));
+    loadTestFile(require.resolve('./classic/_discover_navigation'));
     loadTestFile(require.resolve('./_filters'));
+    loadTestFile(require.resolve('./classic/_filters'));
     loadTestFile(require.resolve('./_size'));
-    loadTestFile(require.resolve('./_date_nanos'));
-    loadTestFile(require.resolve('./_date_nanos_custom_timestamp'));
```

Comment:
Sorry, I think my latest comment offline was misleading ... data nanos test can stay, but just for the Document explorer, no reason to double test with the classic one :)

Reply:
Btw, since you were taking care of the context ones so far, you could definitely finish this one for context, so we can merge, and create a new PR for the main discover ones. keeping the numbers of file changes low is always a good way to get better reviews :)